.github/workflows/ci-app-examples.yml

```diff
@@ -12,7 +12,8 @@ on:
       - "src/lightning_app/**"
       - "tests/tests_app_examples/**"
       - "examples/app_*"
-      - "requirements/app/**"
+      - "requirements/app/*"
+      - "!requirements/app/docs.txt"
```

Comment:
I'm not sure that this works as you expect.

A change to `src/foo/bar` matches `!requirements/app/docs.txt`.
Whereas I believe you want to just exclude this particular file.

You have two options.

a) List all `.txt` files explicitly here without `docs.txt`
b) Use paths-ignore: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths

Same comment applies to all other changes

Reply:
I believe that if it does not work, it would be a bug on their side... :confused:

Reply:
Oops I misread GitHub docs. Resolving